assets/js/grp-home.js

```diff
@@ -2,9 +2,12 @@ import Vue from "vue";
 import '../css/grp-home.css';
 import { BootstrapVue, IconsPlugin } from 'bootstrap-vue';
 import FundingCyclesList from "./vue/components/FundingCyclesList";
+import filters from "./vue/utils/filters";
 
 Vue.use(BootstrapVue);
-
+Vue.filter('truncate', (text, length) => {

```

Comment:
Is there no way to direct reference this?

Reply:
You mean directly include it in the child component?

Reply:
When applying the filter, maybe like this: {{ bla bla | filters.truncate(10) }}
Don't know if that's possible. Maybe that's not the way to do it, because then the child needs to know about the filter name.
Or maybe you can make an alias. Either way, it's the same effect. So this works.